templates/vue/src/store/actions.js

```diff
@@ -223,32 +222,20 @@ export async function getTapestryExport({ dispatch }) {
 
 export async function completeQuestion(
   { commit, dispatch },
-  { answerType, formId, nodeId, questionId }
+  { answerType, answer, nodeId, questionId }
```

Comment:
Again I think this should be ordered better: `nodeId, questionId, answerType, answer`